R/rd-r6.R

```diff
@@ -445,3 +415,16 @@ r6_all_examples <- function(block, methods) {
       )
   }))
 }
+
+first_five <- function(x) {
+  x <- encodeString(x, quote = "`")
+  if (length(x) > 5) x <- c(x[1:5], "...")
+  paste(x, collapse = ", ")
+}
+
+r6_warning <- function(template, block, bad) {
```

Comment:
Can you please make `block` the first argument so it's consistent with `roxy_tag_warning()`?

Reply:
Sure.

Reply:
Done.